src/docs/product/dashboards/custom-dashboards/index.mdx

```diff
@@ -89,4 +89,15 @@ transactions'.
 
 ## Data Set Selection
 
-Select between events and [issues](/product/issues/) data in a widget with the "Data Set" selector. Choosing "Issues" allows you to query using issues filters, such as `is:unresolved` for unresolved issues, and to display and sort by issues fields. This feature only applies to [table](#table-results) visualizations.
+Select between events and [issues](/product/issues/) data in a widget with the "Data Set" selector.
+
+Choosing "All Events" allows you to query and aggregate error and transaction events in the same way you would for a [Discover query](/product/discover-queries/). Some widget examples include:
+- Tracking performance of an endpoint.
+- Throughput by country.
+- Users most affected by errors.
```

Comment:
Since none of these are actually sentences, removing the period.

```suggestion
- Tracking performance of an endpoint
- Throughput by country
- Users most affected by errors
```